apps/frontend/src/app/components/map-review/map-review-list.component.html

```diff
@@ -1,31 +1,37 @@
-@if (!isSubmitter) {
-  <m-map-review-form [map]="map" (reviewPosted)="load.next()" />
-}
-<p-select
-  class="self-end"
-  [(ngModel)]="filter"
-  [options]="Filters"
-  optionLabel="label"
-  optionValue="type"
-  (onChange)="load.next()"
-  appendTo="body"
-/>
-<div [mSpinner]="loading" class="flex flex-col gap-4">
-  @for (review of reviews; track $index) {
-    <m-map-review [review]="review" [map]="map" (updatedOrDeleted)="load.next()" />
-  } @empty {
-    @if (!loading) {
-      <p class="m-4 text-center text-lg italic text-gray-200">No reviews posted</p>
-    }
+@if (localUserService.isLoggedIn) {
+  @if (!isSubmitter) {
+    <m-map-review-form [map]="map" (reviewPosted)="load.next()" />
   }
-</div>
-<p-paginator
-  class="self-end"
-  (onPageChange)="pageChange.next($event)"
-  [first]="first"
-  [rows]="rows"
-  [totalRecords]="totalRecords"
-  [showCurrentPageReport]="true"
-  [alwaysShow]="false"
-  [rowsPerPageOptions]="[5, 10, 20]"
-/>
+  <p-select
+    class="self-end"
+    [(ngModel)]="filter"
+    [options]="Filters"
+    optionLabel="label"
+    optionValue="type"
+    (onChange)="load.next()"
+    appendTo="body"
+  />
+  <div [mSpinner]="loading" class="flex flex-col gap-4">
+    @for (review of reviews; track $index) {
+      <m-map-review [review]="review" [map]="map" (updatedOrDeleted)="load.next()" />
+    } @empty {
+      @if (!loading) {
+        <p class="m-4 text-center text-lg italic text-gray-200">No reviews posted</p>
+      }
+    }
+  </div>
+  <p-paginator
+    class="self-end"
+    (onPageChange)="pageChange.next($event)"
+    [first]="first"
+    [rows]="rows"
+    [totalRecords]="totalRecords"
+    [showCurrentPageReport]="true"
+    [alwaysShow]="false"
+    [rowsPerPageOptions]="[5, 10, 20]"
+  />
+} @else {
+  <p class="text-lg font-bold text-gray-100">
```

Comment:
Nit: bold looks a little strong in your screenshot, I would go with `medium` at most